src/logging.c

```diff
@@ -63,3 +63,17 @@ void log_angular_speed(void)
 	LOG_INFO("%f,%f,%f,%d,%d", target_speed, ideal_speed, angular_speed,
 		 pwm_left, pwm_right);
 }
+
+/**
+ * @brief Log sensors distance: side left, side right, front left and front
+ * right.
```

Comment:
Brief should only be a single line. I'd say "Log all sensor distance readings".

Don't think it is necessary to add an extended description to enumerate the sensors.

Reply:
Right, I copied your single line.